docker/pravega/Dockerfile

```diff
@@ -13,27 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-FROM openjdk:11.0.8-jre-slim
+FROM adoptopenjdk/openjdk11:jre-11.0.11_9-alpine
 
-RUN apt-get update && apt-get install -y -q \
-    rpcbind \
-    nfs-common \
-    python \
```

Comment:
This would affect the entry point script https://github.com/pravega/pravega/blob/56127851f076ba96938a1aae25294ac28d17abb9/docker/pravega/scripts/entrypoint.sh#L31

I do not see this being used when Pravega is being deployed on K8s. 
It does seem to be used on docker-compose. https://github.com/pravega/pravega/blob/a5f5d670b7f2340b8388a9a2b216aa7e5b4d7151/docker/compose/swarm/pravega.yml#L32

Reply:
if you refer to python then it's installed in the new image too. I had to add it exactly for `wait_for` logic. Actually we need to remove `wait_for` as it's a simple one but requires entire python installation.

Reply:
ok.

Reply:
Btw, `WAIT_FOR` is also used in K8s, Pravega Operator sets it for both Controller and Segment Store 
https://github.com/pravega/pravega-operator/blob/master/pkg/controller/pravega/pravega_segmentstore.go#L299